From 439768c4f2e7be5b23cb0902b3002f8c945a9a47 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 11 Aug 2015 18:16:48 +0000
Subject: [PATCH 186/462] MANTIS 0037685 initial: GAL: Application Admin
 Options - Non GUI

added JarMassaiAdminApplication = massai-admin-app.jar
added AdminApplication to build directory list

git-svn-id: svn://localhost/SelfServiceBuild/trunk@278 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 1 +
 SelfServiceBuild/Build/java/build.xml         | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 2ad9ca6..bbf8cce 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -234,6 +234,7 @@
 <property name="JarMassaiCORBASettings"     value="${JarDir}/massai-corba-settings.jar"/>
 <property name="JarMassaiSOAPSettings"      value="${JarDir}/massai-soap-settings.jar"/>
 <property name="JarMassaiAppAdapter"        value="${JarDir}/massai-app-adapter.jar"/>
+<property name="JarMassaiAdminApplication"  value="${JarDir}/massai-admin-app.jar"/>
 <property name="JarMassaiReleaseNotes"      value="${JarDir}/releasenotes.jar"/>
 <property name="JarMassaiStatechart"        value="${JarDir}/massai-statechart.jar"/>
 <property name="JarMatacClient"             value="${JarDir}/MATAC-CLIENT.jar"/>
diff --git a/SelfServiceBuild/Build/java/build.xml b/SelfServiceBuild/Build/java/build.xml
index d3c9739..7ebb052 100644
--- a/SelfServiceBuild/Build/java/build.xml
+++ b/SelfServiceBuild/Build/java/build.xml
@@ -24,7 +24,7 @@
 
     <!-- Definition der Verzeichnisse mit den Java-Modulen ======= -->
     <property name="CommonDirList"   value="${CommonJavaDir}/OpenORB-1.3.1/OpenORB"/>
-    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/ApplicationAdapter,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet"/>
+    <property name="PlatformDirList" value="${PlatformJavaDir}/CUSS,${PlatformJavaDir}/OPI.1.3,${PlatformJavaDir}/OPI_CCV20120201,${PlatformJavaDir}/Tools,${PlatformJavaDir}/MATAC-Client,${PlatformJavaDir}/CussIF,${PlatformJavaDir}/Config,${PlatformJavaDir}/icnTunnelClient,${PlatformJavaDir}/PlatformManager,${PlatformJavaDir}/ApplicationAdapter,${PlatformJavaDir}/AdminApplication,${PlatformJavaDir}/PrintService,${PlatformJavaDir}/CORBA-Settings,${PlatformJavaDir}/SOAP-Settings,${PlatformJavaDir}/CLAApplet"/>
     <property name="AblsDirList"     value="${AblsJavaDir}/KioskProxy,${AblsJavaDir}/TunnelServer"/>
     <property name="AppDirList"      value=""/>
 
-- 
2.41.0.windows.1

